addons/product_expiry/models/production_lot.py

```diff
@@ -15,7 +15,7 @@ class StockProductionLot(models.Model):
     removal_date = fields.Datetime(string='Removal Date',
         help='This is the date on which the goods with this Serial Number should be removed from the stock.')
     alert_date = fields.Datetime(string='Alert Date',
-        help="This is the date on which an alert should be notified about the goods with this Serial Number.")
+        help='Date used to determined what are the expired lots and serial numbers using the filter "Expiration Alerts".')
```

Comment:
spelling mistakes : here is the right string
Date used to determine the expired lots and serial numbers using the filter "Expiration Alerts"